.idea/vcs.xml

```diff
@@ -2,6 +2,5 @@
 <project version="4">
   <component name="VcsDirectoryMappings">
     <mapping directory="$PROJECT_DIR$" vcs="Git" />
-    <mapping directory="$PROJECT_DIR$/IT3048" vcs="Git" />
```

Comment:
This VCS mapping was causing errors in my android studio, and as far as I can tell it should not be  here. The file it was pointing at was some sort of weird symbolic link to a commit so I deleted that too.